core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -119,22 +119,22 @@ class SparkContext(config: SparkConf) extends Logging {
   /**
    * Alternative constructor that allows setting common Spark properties directly
    *
-   * @param master Cluster URL to connect to (e.g. mesos://host:port, spark://host:port, local[4]).
-   * @param appName A name for your application, to display on the cluster web UI
-   * @param conf a [[org.apache.spark.SparkConf]] object specifying other Spark parameters
+   * @param master cluster URL to connect to (e.g. mesos://host:port, spark://host:port, local[4]).
+   * @param appName a name for your application, to display on the cluster web UI
+   * @param conf a `org.apache.spark.SparkConf` object specifying other Spark parameters
```

Comment:
This should render fine if I remember correctly although I am fine with either way but I would like to know the reason to make this wrapped with backticks if there is.